### ISSUE TITLE
how this component play rtmp stream?

### ISSUE THREAD

**Issue by x313253805** [mingxinstar/react-hls#4]
I try to play rtmp stream in my project,this component is well with m3u8 stream.can this component support rtmp stream as well?

**Comment by MCDELTAT** [comment]
Yes it will. See the first paragraph of the [README](https://github.com/mingxinstar/react-hls/blob/master/README.md) which says:

`react-hls` is a simple hls/rtmp live stream player.
It will use [hls.js](https://github.com/dailymotion/hls.js) to play your hls live stream if your browser support `html 5 video` and `MediaSource Extension`. Otherwise it will downgrade to `Flash` play your `rtmp` live stream.

**Comment by xiaomengheiheihei** [comment]
Can you give a simple example?

**Comment by coder054** [comment]
> 
> 
> Yes it will. See the first paragraph of the [README](https://github.com/mingxinstar/react-hls/blob/master/README.md) which says:
> 
> `react-hls` is a simple hls/rtmp live stream player.
> It will use [hls.js](https://github.com/dailymotion/hls.js) to play your hls live stream if your browser support `html 5 video` and `MediaSource Extension`. Otherwise it will downgrade to `Flash` play your `rtmp` live stream.

In the to do list of Readme, The work for RTMP is not check(not done)

**Comment by MCDELTAT** [comment]
@coder054 Really had to downvote something from a year ago huh? I might have tested this package and gotten it working but don't remember how. IMO I haven't recently looked for any packages that do all 3 (HLS, RTMP, MPEG-DASH) but your best bet is sticking with the video.js family of tech. Here's a gist of what I did 2 years ago to get it working:

https://gist.github.com/MCDELTAT/91829acd6752815dc4e88c9c0120a315

Basically, I have a component for HLS streaming, one regular video.js Component for RTMP, and then a Component that can destroy one player and create the other.

Again that was 2.5 years ago and a lot has changed so dunno.
